src/components/RegisterForm/RegisterForm.jsx

```diff
@@ -77,100 +93,128 @@ const RegisterForm = () => {
-          placeholder="Enter email"
-          color={{
-            error: formik.errors.userEmail,
-            touched: formik.touched.userEmail,
-          }}
-        />
-
-        {formik.errors.userEmail ? (
-          <Error>{formik.errors.userEmail}</Error>
-        ) : !formik.errors.userEmail && formik.touched.userEmail ? (
-          <Success>Field is not empty</Success>
-        ) : null}
-      </ItemWrapp>
-
-      <ItemWrapp>
-        <StyledLabel
-          color={{
-            error: formik.errors.password,
-            touched: formik.touched.password,
-          }}
-          htmlFor="password"
-        >
-          Password
-        </StyledLabel>
-        <StyledInput
-          id="password"
-          name="password"
-          type="password"
-          onChange={formik.handleChange}
-          value={formik.values.password}
-          placeholder="Enter password"
-          color={{
-            error: formik.errors.password,
-            touched: formik.touched.password,
-          }}
-          style={{ marginBottom: 32 }}
-        />
-
-        {formik.errors.password && formik.touched.password ? (
-          <Error>{formik.errors.password}</Error>
-        ) : !formik.errors.password && formik.touched.password ? (
-          <Success>Field is not empty</Success>
-        ) : null}
-      </ItemWrapp>
-
-      <SignUpBtn type="submit">Sign Up</SignUpBtn>
-    </form>
+            {formik.errors.password && formik.touched.password ? (
+              <Error>{formik.errors.password}</Error>
+            ) : !formik.errors.password && formik.touched.password ? (
+              <Success>Field is not empty</Success>
+            ) : null}
+          </ItemWrapp>
+          <ItemWrapp>
+            <StyledLabel
+              color={{
+                error: formik.errors.repeatPassword,
+                touched: formik.touched.repeatPassword,
+              }}
+              htmlFor="repeatPassword"
+            />
+            <StyledInput
+              id="repeatPassword"
+              name="repeatPassword"
+              type="password"
+              onChange={formik.handleChange}
+              value={formik.values.repeatPassword}
+              placeholder="Password again"
+              color={{
+                error: formik.errors.repeatPassword,
+                touched: formik.touched.repeatPassword,
+              }}
+              style={{ marginBottom: 32 }}
+            />
+            {formik.errors.repeatPassword && formik.touched.repeatPassword && (
+              <Error>{formik.errors.repeatPassword}</Error>
+            )}
+          </ItemWrapp>
+          <SignUpBtn type="submit">Create</SignUpBtn>
+          <Separator data-content="or" />
+          <ButtonBlock>
+            <ButtonGoogle type="button" />
+            <ButtonFacebook type="button" />
+          </ButtonBlock>
+        </SignInFormStyles>
+      </RegisterFormContainer>
+    </Background>
```

Comment:
Ideally component should not contain more then 60- 80 rows. If it is bigger then you should do a refactor and split it out for smaller peases.